source/standards/accounts-with-third-parties.html.md.erb

```diff
@@ -0,0 +1,146 @@
+  separate username and password
+- Prefer using G-Suite as a single sign-on provider over any other
+  provider
+    - this is because G-Suite is a core part of the leaver's process and
+      so we have high confidence this will be revoked when somebody
+      leaves
+    - Sometimes GitHub may be preferable as sign-on provider, for
+      services that can use GitHub’s team structure to manage
+      permissions. For example, travis does this.
+- Ensure that you have a documented process for removing individual's
+  access when they leave.
+    - If you use G-Suite then their access will be revoked when their
+      G-Suite access is revoked.  However, you may still wish to clean
+      up the individual's account with the third-party service.
+
+### Services that only provide individual accounts
+
+For services which do not support organisation accounts, there is no
+choice but to share credentials.  This carries a number of problems:
+
+- It's not always easy to know who has access to a service,
+  particularly an occasionally-accessed service
+- If the last person with access leaves, we can lose access altogether
+- If the account is registered with an individual's email address, and
+  that person leaves, a number of features may be lost (such as
+  important notifications or "lost password" functionality)
+
+These recommendations try to reduce the risks of these adverse
+outcomes:
+
+- Create an account using a Google group email address rather than an
+  individual email address to avoid losing access to the account when
+  people leave (see notes below on using shared Google groups).
+- Store the password in your team's shared credential store.
+- If the account supports two-factor authentication, ensure each team
+  member has access to a second factor for the account
+- Ensure you have a documented process for removing individual's
+  access to the Google group and the credential store and for rotating
+  the credentials including any API keys and multi-factor
+  authentication.
+
+## Notes on using shared Google Groups
+
+Ensure that:
+
+- the group is able to receive emails from outside of the organisation by 
+  setting 'permission to post' to 'Public'
+- only users who should be able to access the service are able to view the
+  posts by setting 'permission to view topics' to 'all members of the group'
+
+## Recommendations for specific services:
+
+### GitHub
+
+Key recommendation: **use separate accounts per individual**.
+
+- Individuals may have existing GitHub accounts when they join GDS; we
+  can use those rather than creating a separate GDS account
+    - However, this means it is doubly important to ensure that
+      individuals are removed from our GitHub organisation as part of
+      the leaver's process.
+- Each team should have someone (usually a tech lead) who is an owner
+  and can grant/revoke access
+- CI users and dashboards such as fourth wall should use scoped OAuth
```

Comment:
Use caps: 'Fourth Wall'

Reply:
amended this and linked to Fourth Wall - https://github.com/alphagov/fourth-wall